### PR TITLE
feat(TX-231): release new order breakdown

### PR DESCRIPTION
**Pull request by rquartararo**
[TX-231](https://artsyproduct.atlassian.net/browse/TX-231)

This PR releases the [new order breakdown](https://artsyproduct.atlassian.net/browse/TX-90) by taking it out from behind the Avalara Phase 2 feature flag.

### CODE REVIEW

File: src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx [hunk 1]
```diff
@@ -80,15 +76,13 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
         <Text variant="sm" color="black60">
           {shippingOrigin}
         </Text>
-        {user && userHasLabFeature(user, "Avalara Phase 2") && (
```
Comment (starsirius) [inline src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx:83]: Is the lab feature still used after this PR? If not, we can retire it!

**Review by starsirius — APPROVED** [review]
Exciting!! 🎉

File: src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx (hunk 1 above)
Comment (rquartararo) [reply src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx:83]: @starsirius Confirmed with Roberta this will be staying behind the flag.

Comment (starsirius) [reply src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx:83]: Ohhh yeah you're right. Let's keep it! Thanks for double checking!

**Pull request merged**